src/components/parameters/ParameterUnits.svelte

```diff
@@ -0,0 +1,26 @@
+<svelte:options immutable={true} />
+
+<script lang="ts">
+  import { tooltip } from '../../utilities/tooltip';
+
+  export let unit: string | undefined = undefined;
+</script>
+
+{#if unit}
+  <span class="parameter-units" use:tooltip={{ content: unit }}>{unit}</span>
```

Comment:
I'm assuming expectation here is that units will be a few characters at max? Setting this example to a longer string does this (which makes sense with the styling you've done) but was just curious if users are going to actually stick to short unit names.
<img width="538" alt="image" src="https://github.com/NASA-AMMOS/aerie-ui/assets/4419607/dca7c577-5269-4601-b18a-60b09ed474d3">

Additionally - are longer unit descriptions and parameter descriptions in the development pipeline? Seeing those in the mockups and _kind of_ in the db (some sort of ```..."banannotation":{"value":"Specifies the size of bite to take"...``` thing that might just be placeholder?)

Reply:
i don't think users will always be sticking with short units.

Yeah, unit descriptions are not yet available, but I believe the goal is to have them at some point. The "banannotation" thing is part of the `metadata`, which is essentially arbitrary at this point aside from `unit`. We'd need to come up with some place to show the metadata (could be in the tooltip), but that should be a separate effort probably.